src/main/Race.java

```diff
@@ -0,0 +1,29 @@
+package main;
+
+public class Race {
+    int round;
+    Team firstTeam;
+    Team secondTeam;
+    String time;
```

Comment:
Use lombok for getters, setters, constructors. Also your fields should have private modificator